components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/utils/AdminAttributeUtilTest.java

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2017, WSO2 LLC. (http://www.wso2.org).
```

Comment:
```suggestion
 * Copyright (c) 2017-2024, WSO2 LLC. (http://www.wso2.org).
```

Reply:
need to update the year range and LLC part in the copy right license in other changed files